src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Node for teleop-related callback functions
+
+from math import copysign
+from enum import IntEnum
+import rospy as ros
+from sensor_msgs.msg import Joy, JointState
+from geometry_msgs.msg import Twist
+from mrover.msg import Chassis, RAOpenLoopCmd, HandCmd
+
+
+def quadratic(val):
+    return copysign(val**2, val)
+
+
+def deadzone(magnitude, threshold):
+    temp_mag = abs(magnitude)
+    if temp_mag <= threshold:
+        temp_mag = 0
+    else:
+        temp_mag = (temp_mag - threshold) / (1 - threshold)
+
+    return copysign(temp_mag, magnitude)
+
+
+class Drive:
+    class LogitechAxes(IntEnum):
+        left_right = 0
+        forward_back = 1
+        twist = 2
+        dampen = 3
+        pan = 4
+        tilt = 5
+
+    # Constants for diff drive
+    TRACK_RADIUS = 10.0  # meter
```

Comment:
this prob belongs in ros param server

Reply:
both track_radius and wheel_radius (i'm actually not sure what a track radius is though)